components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -3294,7 +3219,8 @@ private API importSOAPToRESTAPI(InputStream fileInputStream, Attachment fileDeta
                     throw new APIManagementException(ExceptionCodes.UNSUPPORTED_WSDL_FILE_EXTENSION);
                 }
             }
-            String updatedSwagger = updateSwagger(createdApi.getUUID(), swaggerStr);
+            //TODO-ORG
```

Comment:
SOAP to REST is not required for the initial implementation.

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9684/commits/cc8517b20f7dbec93569cf9ab02809b6b9148cbe